index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jim Ntare | Software Engineer</title>
+    <link rel="stylesheet" href="./build/styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="nav_bar flex">
+        <div class="logo">My Logo</div>
+        <div class="nav_items flex">
+          <div class="nav_item">Portfolio</div>
+          <div class="nav_item">About</div>
+          <div class="nav_item">Contact</div>
+        </div>
+        <div class="hamburger" onclick="toggle(this)">
+          <div class="burger-one"></div>
+          <div class="burger-two"></div>
+          <div class="burger-three"></div>
+        </div>
```

Comment:
- [ ] [OPTIONAL] I highly recommend you make this change. I think it's better to use the HTML semantic `nav` tag instead of div. To make your webpage more semantically correct.